src/query/api/v1/httpd/handler.go

```diff
@@ -96,14 +99,20 @@ func NewHandler(
 ) (*Handler, error) {
 	r := mux.NewRouter()
 
-	// apply middleware. Just CORS for now, but we could add more here as needed.
-	withMiddleware := &cors.CORSHandler{
+	// apply middleware.
+	var withMiddleware http.Handler = &cors.Handler{
```

Comment:
As a nit, I think this may read slightly cleaner as 
```
corsMiddleware := &cors.Handler{...}
...
withMiddleware := nethttp.Middleware(opentracing.GlobalTracer(), corsMiddleware, ...)